package.json

```diff
@@ -18,11 +23,26 @@
     "hof-template-mixins": "^5.0.1",
     "hof-theme-govuk": "^4.3.0",
     "hof-util-countries": "^1.0.0",
-    "typeahead-aria": "^1.0.4"
+    "jquery": "^3.6.0",
```

Comment:
Where is jQuery required as a new dependency?

Reply:
It was throwing a linting error because jquery was required within assets/js/index.js but not included in the dependencies.  This file was written by the Marvel team and not to do with our integration. Not too sure why it was working without it before and I suspect the code in that file might be redundant. Will come back and look at it.